### PR TITLE
SDI renamed from IESD

### PR DESCRIPTION
**Pull request by sungeunbae**
Small change to make qcore aware of the name change from IESD to SDI.
Unfortunately, black made a lot more changes that I actually did.

### CODE REVIEW

File: qcore/geo.py [hunk 1]
```diff
@@ -465,6 +465,58 @@ def avg_wbearing(angles):
...
+        lon=bottom_mid[0],
+        distance=x_shift,
+        bearing=(bottom_mid_bearing + 90) % 360,
+    )[::-1]
+
+    # at this point we have a perfect square (by corner distance)
+    # c1 -> c4 == c2 -> c3 (right == left), c1 -> c2 == c3 -> c4 (top == bottom)
+    return c1, c2, c3, c4
+
+
```
Comment (jpa198) [inline qcore/geo.py:519]: Wasn't this already in here? Also it's not related to SDI

**Review by jasonmotha — COMMENTED** [review]
No changes requested. But waiting on James's comment  + merge conflict

File: qcore/im.py [hunk 2]
```diff
@@ -153,16 +155,18 @@ def get_unit(self):
...
             return "cm/s"
         elif self.name in [IMEnum.CAV, IMEnum.FAS]:
-            return "gs"  # FAS could also be cm/s depending on calculation / implementation
+            return (
+                "gs"  # FAS could also be cm/s depending on calculation / implementation
+            )
         elif self.name in [IMEnum.Ds575, IMEnum.Ds595, IMEnum.Ds2080]:
             return "s"
-        elif self.name in [IMEnum.MMI, IMEnum.IESDR]:
+        elif self.name in [IMEnum.MMI, IMEnum.SDI]:
```
Comment (jpa198) [inline qcore/im.py:163]: SDI is displacement, so should have units of m or cm?

File: qcore/geo.py (hunk 1 above)
Comment (jpa198) [reply qcore/geo.py:519]: This will probably be resolved when master is merged in...

File: qcore/im.py (hunk 2 above)
Comment (sungeunbae) [reply qcore/im.py]: Well spotted. Thanks

File: qcore/geo.py (hunk 1 above)
Comment (sungeunbae) [reply qcore/geo.py:519]: Yes, once the PR is merged in (vm_gen_refactor) - it'll be all cleaned up.

File: qcore/im.py (hunk 2 above)
Comment (sungeunbae) [reply qcore/im.py]: unit of cm, it is.

File: qcore/im.py [hunk 3]
```diff
@@ -153,23 +156,27 @@ def get_unit(self):
...
         elif self.name in [IMEnum.CAV, IMEnum.FAS]:
-            return "gs"  # FAS could also be cm/s depending on calculation / implementation
+            return (
+                "gs"  # FAS could also be cm/s depending on calculation / implementation
+            )
         elif self.name in [IMEnum.Ds575, IMEnum.Ds595, IMEnum.Ds2080]:
             return "s"
-        elif self.name in [IMEnum.MMI, IMEnum.IESDR]:
+        elif self.name in [IMEnum.MMI]:
             return ""  # MMI is dimensionless & Ratios are dimensionless
```
Comment (jasonmotha) [inline qcore/im.py:165]: Update this comment too. As there are no longer any ratios here.

File: qcore/im.py [hunk 4]
```diff
@@ -129,7 +129,10 @@ class IM:
     def __post_init__(self):
         if not isinstance(self.name, IMEnum):
             self.name = IMEnum[self.name]
-        if self.component is not None and not isinstance(self.component, constants.Components):
+
+        if self.component is not None and not isinstance(
+            self.component, constants.Components
+        ):
             constants.Components.from_str(self.component)
```
Comment (jasonmotha) [inline qcore/im.py:136]: While you are editing this are you able to fix my bug?

`self.component = constants.Components.from_str(self.component)`

File: qcore/im.py (hunk 3 above)
Comment (sungeunbae) [reply qcore/im.py]: Done. Just removed "Ratios are dimensionless" bit.

File: qcore/im.py (hunk 4 above)
Comment (sungeunbae) [reply qcore/im.py]: Done.

**Review by jasonmotha — APPROVED** [review]

**Review by jpa198 — APPROVED** [review]

**Pull request merged**